lib/takeaway.rb

```diff
@@ -0,0 +1,18 @@
+class Takeaway
+  def initialize(menu:menu, order:nil)
```

Comment:
Nice use of required keyword arguments. In `menu:menu`, you're saying that the default value of the thing on the left is the thing on the right. But the thing on the right hasn't been initialized, so this is effectively the same as `menu:nil`.